<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/resource.qrc">
    <normaloff>:/icon/icons/wescon_logo.png</normaloff>:/icon/icons/wescon_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newProjectAction"/>
    <addaction name="importProjectAction"/>
    <addaction name="exportProjectAction"/>
    <addaction name="separator"/>
    <addaction name="saveProjectAction"/>
    <addaction name="takeEffectAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="separator"/>
    <addaction name="addDeviceAction"/>
    <addaction name="deleteDeviceAction"/>
    <addaction name="separator"/>
    <addaction name="addFormulaAction"/>
    <addaction name="deleteFormulaAction"/>
    <addaction name="separator"/>
    <addaction name="addAlarmRuleAction"/>
    <addaction name="deleteAlarmRuleAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newProjectAction"/>
   <addaction name="importProjectAction"/>
   <addaction name="exportProjectAction"/>
   <addaction name="separator"/>
   <addaction name="saveProjectAction"/>
   <addaction name="takeEffectAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newProjectAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/new_file.png</normaloff>:/icon/icons/new_file.png</iconset>
   </property>
   <property name="text">
    <string>新建项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="importProjectAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/import.png</normaloff>:/icon/icons/import.png</iconset>
   </property>
   <property name="text">
    <string>还原项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="exportProjectAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/export.png</normaloff>:/icon/icons/export.png</iconset>
   </property>
   <property name="text">
    <string>备份项目</string>
   </property>
   <property name="toolTip">
    <string>备份项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="addDeviceAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/device.png</normaloff>:/icon/icons/device.png</iconset>
   </property>
   <property name="text">
    <string>添加装置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="deleteDeviceAction">
   <property name="text">
    <string>删除装置</string>
   </property>
  </action>
  <action name="addFormulaAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/calculator.png</normaloff>:/icon/icons/calculator.png</iconset>
   </property>
   <property name="text">
    <string>添加公式</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="deleteFormulaAction">
   <property name="text">
    <string>删除公式</string>
   </property>
  </action>
  <action name="addAlarmRuleAction">
   <property name="icon">
    <iconset>
     <normaloff>../resource/icons/alarm.png</normaloff>../resource/icons/alarm.png</iconset>
   </property>
   <property name="text">
    <string>添加告警规则</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="deleteAlarmRuleAction">
   <property name="text">
    <string>删除告警规则</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/exit.png</normaloff>:/icon/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出应用</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="saveProjectAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/save_as.png</normaloff>:/icon/icons/save_as.png</iconset>
   </property>
   <property name="text">
    <string>保存项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="takeEffectAction">
   <property name="icon">
    <iconset resource="../resource/resource.qrc">
     <normaloff>:/icon/icons/exec.png</normaloff>:/icon/icons/exec.png</iconset>
   </property>
   <property name="text">
    <string>配置生效</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
